human-controls/src/DisplayController.cpp

```diff
@@ -0,0 +1,29 @@
+#include "DisplayController.h"
+
+DisplayController::DisplayController(unsigned int displayAddress, unsigned int displayLen, unsigned int displayWidth)
+{
+    this->lcd = new LiquidCrystal_I2C(displayAddress, displayLen, displayWidth);
```

Comment:
This can be removed in favor of `DisplayController::DisplayController(...) : lcd(displayAddress, displayLen, displayWidth)`

Reply:
Done